src/Artsy/Router/buildServerApp.tsx

```diff
@@ -96,6 +99,7 @@ export function buildServerApp(config: RouterConfig): Promise<Resolve> {
           <App data={relayData} loadableState={loadableState} {...props} />
         ),
         status,
+        headTags,
```

Comment:
`headTags` here is return in the resolve of `buildServerApp` which can be used in `server.js` in force to actually provide the head markup via SSR. See https://github.com/artsy/force/pull/2881.